app/routes/index.js

```diff
@@ -0,0 +1,11 @@
+
+//chun added this
```

Comment:
Please remove this line of code... We don't need to know who added which lines. There are commands like `git blame` which will tell us exactly who made which changes.